packages/cadl-ranch-specs/http/resiliency/dev-driven/main.cadl

```diff
@@ -101,7 +102,7 @@ This method requires to write 2 tests.
 @route("/customization/paging/{mode}")
 @doc("Get pages that you will either return to users in pages of raw bodies, or pages of models following growup.")
 @OpenAPI.operationId("getPages") // Do NOT read this from your CADL emitter, this is here only for Swagger generation accuracy
-op getPages is ResourceList<Product, ModeParameter>;
+op getPages is Page<Product>;
```

Comment:
Will CI try to compile cadl files? When I try to compile this file, I got this error: Operation can only reuse the signature of another operation.

Reply:
I believe it does, I didn't get that error, but it was failing the validate scenarios test. When I switched to ResourceList it seems the issue is now fixed